src/types/types.ts

```diff
@@ -207,6 +223,8 @@ export type KeywordListResponse = {
 export type LandingPage = {
    __typename?: 'LandingPage',
   id: Scalars['ID'],
+  topBanner?: Maybe<BannerPage>,
```

Comment:
What is this topBanner? Is it going to be used later?

Reply:
it is the banner that is already show in frontend`

Reply:
it contains the data that is used as the top banner at the moment

Reply:
Now that you add new banner you should use these fields for the topBanner that already exists

Reply:
after that, topBanner properties from this root level will be removed (the deprecated fields=

Reply:
If I now understand this correctly